packages/logger/src/index.ts

```diff
@@ -58,18 +62,27 @@ const getNamespace = (namespace: string): string => {
 
 // tslint:disable-next-line: variable-name
 export const NSlogger = (namespace: string = ''): ILogger => {
-  const namespaceModified = getNamespace(namespace);
-
   Object.assign(instance.defaultMeta, {
     ...instance.defaultMeta,
-    namespace: namespaceModified,
+    namespace: getNamespace(namespace),
   });
 
+  const params = (msg, severity) =>
+    ENV.nodeEnv !== 'development' ? [...msg, { severity, component: 'arbitrary-property' }] : msg;
```

Comment:
To be able to filter logs by severity on GCP I had to log this JSON as well. The only issue is that it would show up when developing locally. So I made it so the JSON only logs when not in a dev environment.

Reply:
I added the severity property to the `gcpLogEntryFormat` function and it works!